tests/test_edge_cases.py

```diff
@@ -497,7 +497,7 @@ class Model(BaseModel):
 
 def test_list_unions():
     class Model(BaseModel):
-        v: List[Union[int, str]] = ...
+        v: List[Union[int, int]] = ...
```

Comment:
I would keep the union of int and str, and try to do an unordered list comparison (a la https://docs.python.org/3.5/library/unittest.html#unittest.TestCase.assertCountEqual but for pytest, don't know if it exists)

Reply:
Oops yes this annotation modification was a mistake.